### DISCUSSION TITLE
About the ui mod

### DISCUSSION THREAD

**Discussion by ghost** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#50] [Ideas]
I once mentioned a ui change someone was making - I found even better. On April 19 on tumblr @justmiha97 posted her version of the ui mod. (https://justmiha97.tumblr.com/post/681997997900431360/sims-1-alternative-ui), but there are flaws there that could have been changed. Would you like to work on improving this ui mod? (I asked, the author allows you to change his stuff, just the main thing is to specify the credits).

**Comment by ghost** [comment]
I have a FarX program to help edit Uigraphics.far as well as ideas on how to improve the panels. I would like to try this in July. If you are interested, we can discuss it on telegram or somewhere else, if you are not interested, please let me know.

**Comment by FaithBeam** [comment]
Sorry for the late response, I had to think about what my plans are for this program. I'm going to say no because I'd like to focus on other projects I'm working on. I'll still release bug fixes, but no new features.

**Reply by ghost** [reply]
Thanks for the answer
